src/nadzoring/network_base/whois_lookup.py

```
@@ -44,43 +44,32 @@ def _is_ip(target: str) -> bool:
-        logger.exception(
-            "WHOIS lookup failed for %s.\n\n"
-            "Possible fixes:\n"
-            "  • Ensure 'whois' is installed:\n"
-            "    - Ubuntu/Debian: sudo apt install whois\n"
-            "    - macOS: brew install whois\n"
-            "    - RHEL/Fedora: sudo dnf install whois\n"
-            "  • Check internet connectivity",
-            target,
-        )
-        return None
+    return check_output(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)